uint8 token
string name
---
string token